packages/@ember/-internals/glimmer/lib/component-managers/custom.ts

```diff
@@ -176,6 +176,29 @@ export default class CustomComponentManager<ComponentInstance>
 
             return ref.value();
           },
+
+          has(_target, prop) {
+            assert('args can only be strings', typeof prop === 'string');
```

Comment:
I think this is slightly incorrect wording, AFAICT the arg _key_ can only be a string (but the value can be anything), right?